.gitignore

```diff
@@ -1,2 +1,4 @@
-_build/
```

Comment:
Not that this matters, but does your editor remove the `/` somehow?

Reply:
esy creates `_build` symlink instead of `_build` dir, removing `/` was needed to make `git` ignore it.

Reply:
Ah, thanks.